app/assets/web-dev-iframe.html

```diff
@@ -10,16 +10,26 @@
 
     <script src="https://cdn.jsdelivr.net/lodash/4.13.1/lodash.min.js" integrity="sha256-8SeyqJ7ZAZx8WnIgP/bgK6LGIjKjhojNPHSMV/fo29Y=" crossorigin="anonymous"></script>
 
+    <!-- TODO: Does jquery font loading affect china customers? -->
     <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
 
     <script src="/javascripts/web-dev-listener.js"></script>
     
     <script src="/javascripts/app/vendor/aether-html.js"></script>
-
-    <style>
-      @import 'https://fonts.googleapis.com/css?family=Holtwood+One+SC';
+    <script>
       /* Import that font for demoing web-dev levels until @import ordering bug is fixed */
-
+      const head = document.head;
```

Comment:
(Mostly a question for my own knowledge - I don't think any changes are necessary for this PR)

Could this be a Jade template that dynamically selects this based on build config?  Or is `features.chinaUx` only available at runtime?

Reply:
It could. This is kind of a low cost change to see if this will work for the html iframe.
This iframe is only aware of the environment at runtime,  but we could definitely build it at build time.

This is the code that is also most likely to not work. I will need someone in china to test this particular code path manually.